docs/how-to/datapath-only-mode.rst

```diff
@@ -0,0 +1,112 @@
+==================
+Datapath-only Mode
+==================
+
+This is a mode of operation in which MicroOVN cluster runs only ``switch`` and
+``chassis`` service, with ``central`` service being provided externally by another
+MicroOVN cluster or by some other OVN deployment.
+
+Cluster setup
+-------------
+
+We will start by deploying the MicroOVN cluster without the ``central`` service. Our
+hosts that run MicroOVN are called ``node-1``, ``node-2`` and ``node-3``. Setup of
+the external OVN central cluster is not covered by this how-to. For the sake of this
+guide we assume that there are OVN Southbound and OVN Northbound services running on
+their default ports on IPs:
+
+* 10.0.0.1
+* 10.0.0.2
+* 10.0.0.3
+
+.. note::
+
+   If you already have MicroOVN deployed, see
+   :doc:`Service Control </how-to/service-control>` about how to disable ``central``
+   service on the running nodes, and :doc:`MicroOVN services </reference/services>`
+   regarding implications of removing all ``central`` services on the running cluster.
+
+Obtain CA certificate and private key for the MicroOVN
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Because the MicroOVN enforces encrypted communication between OVN services, we need
+to ensure that OVN services running on the MicroOVN cluster and on the external OVN
+central cluster are mutually trusted.
+
+Obtain intermediate CA certificate and private key from the CA that issued certificates
+for the external OVN central cluster and place it in the ``/var/snap/microovn/common/``
+on the ``node-1``, under names ``ca.cert`` and ``ca.key``.
+
+Bootstrap MicroOVN cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First we bootstrap the MicroOVN cluster on the ``node-1`` and generate tokens, so that
+``node-2`` and ``node-3`` can join.
+
+.. code-block::
+
+   $ sudo microovn init
```

Comment:
No terminal prefix in code-blocks, make two code-blocks instead, one with the command, another with the output/interaction.